### PR TITLE
Add new papers in application, incentive and survey of VFL

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**
Add a new **DMBD 2021** paper: Feature Map Transfer: Vertical Federated Learning for CNN Models. This paper demonstrate the image classification task in the vertical federated learning setting where each participant holds incomplete image pieces of all samples and  different from traditional federated learning algorithms sharing model parameters in each training iteration,
VFedConv shares hidden feature maps. Each client creates a local feature extractor and transmits the extracted feature maps to the server to protect raw data.It is a awesome application of VFL in CV.

Add a new **COMPSAC2022** paper: A Mobility Forecasting Framework with Vertical Federated Learning. This paper proposes a mobility vertical federated forecasting (MVFF) framework that allows the learning process to be jointly conducted over vertically partitioned data belonging to multiple organizations when existing forecasting approaches usually adopt frameworks with a centralized mobility data holder.It is a awesome application of VFL in smart citiy.

Add a new **Wireless Communications and Mobile Computing 2021** paper: Lstfcfedlear: a LSTM-FC with vertical federated learning network for fault prediction. This paper  propose a vertical federated learning framework based on LSTM fault classification network (LstFcFedLear) which not only can predict when IoT equipment will fail in the future and then provide what measures should be used, but also encrypt and integrate the data on the entire firefighting IoT platform to form a new dataset and to protect raw data. It is a awesome application of VFL.

Add a new **NeurIPS'22** paper: Fairvfl: A fair vertical federated learning framework with contrastive adversarial learning. This paper propose a method based adversarial learning which can improve the fairness of VFL models in a privacy-preserving way.

Add a new **Proceedings of the 8th International Conference on Computing and Artificial Intelligence. 2022**: Multi-Participant Vertical Federated Learning Based Time Series Prediction. This paper demonstrate a application of time series scenario. They provide several methods to achieve better performance on these separated features. Two of their models has shown promising results compared with local training. Also, they add the PSA protocol into training to guarantee the safety of private information.It is a awesome application of VFL in time series prediction.

Add a new **ICDE2022** paper: Efficient Participant Contribution Evaluation for Horizontal and Vertical Federated Learning. This paper propose a highly efficient approach, named DIG-FL, to estimate the Shapley value of each participant without any model retraining. It's worth noting that their approach is applicable to both vertical federated learning (VFL) and horizontal federated learning (HFL). In addition, they propose a DIG-FL based reweight mechanism to improve the model training in terms of accuracy and convergence speed by dynamically adjusting the weights of participants according to their per-epoch contributions, and theoretically analyze the convergence speed. 

Add a new survey paper: Vertical Federated Learning: A Structured Literature Review. This paper provides a structured review on the current research trends on VFL. Following a structured protocol for review ensures that the review is comprehensive, rigorous, and transparent, and that the results are easily accessible and understandable to the intended audience. The review
was planned, conducted, and reported in accordance with the SLR process proposed by Armitage and Keeble-Allen (2008).

### CODE REVIEW

File: docs/awesome-pets/papers/applications/ppml/fl/application.md [hunk 1]
```diff
@@ -157,6 +159,8 @@ Keywords: Split Learning + Sparsification
 
 - [Exploiting Unlabeled Data in Smart Cities using Federated Learning](https://arxiv.org/pdf/2001.04030.pdf)
 
+- [A Mobility Forecasting Framework with Vertical Federated Learning](https://ieeexplore.ieee.org/abstract/document/9842629)
+
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/application.md:163]: It is not recommended to include C-level conferences in the awesome-list

Comment (Mingbo-Lee) [reply docs/awesome-pets/papers/applications/ppml/fl/application.md]: Thank you for your suggestion.

File: docs/awesome-pets/papers/applications/ppml/fl/incentive.md [hunk 2]
```diff
@@ -12,6 +12,8 @@
 
 - [Fedcoin: A peer-to-peer payment system for federated learning](https://arxiv.org/pdf/2002.11711.pdf)
 
+- [Efficient Participant Contribution Evaluation for Horizontal and Vertical Federated Learning](https://ieeexplore.ieee.org/document/9835159)
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/incentive.md:15]: CCF C is not recommended

File: docs/awesome-pets/papers/applications/ppml/fl/survey.md [hunk 3]
```diff
@@ -9,6 +9,8 @@
 
 - [Vertical Federated Learning](https://arxiv.org/pdf/2211.12814.pdf)
 
+- [Vertical Federated Learning: A Structured Literature Review](https://arxiv.org/pdf/2212.00622.pdf)
+
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/survey.md:13]: Please recommend articles that are acccepted by CCF B or above.

**Comment by Mingbo-Lee** [comment]
Hi！ @zhangxingmeng Thank you for your review and suggestion. I'm busy for my exam recently and I will deal with the problems in several days.
please wait me patiently.

File: docs/awesome-pets/papers/applications/ppml/fl/application.md [hunk 4]
```diff
@@ -36,6 +36,8 @@
 
 - [FedVision: An Online Visual Object Detection Platform Powered by Federated Learning](https://arxiv.org/pdf/2001.06202.pdf)
 
+- [Feature Map Transfer: Vertical Federated Learning for CNN Models](https://link.springer.com/content/pdf/10.1007/978-981-16-7502-7_4.pdf)
```
[the remark this answers is not in the document]
Comment (Mingbo-Lee) [reply docs/awesome-pets/papers/applications/ppml/fl/application.md:39]: This paper demonstrate the image classification task in the vertical federated learning setting where each participant holds incomplete image pieces of all samples and different from traditional federated learning algorithms sharing model parameters in each training iteration, VFedConv shares hidden feature maps. Each client creates a local feature extractor and transmits the extracted feature maps to the server to protect raw data.It is a awesome application of VFL in CV.

File: docs/awesome-pets/papers/applications/ppml/fl/application.md [hunk 5]
```diff
@@ -260,6 +265,7 @@ Citation: 0
 - [GRAFFL: Gradient-free Federated Learning of a Bayesian Generative Model](https://arxiv.org/pdf/1910.08489.pdf)
 
 
+
```
[the remark this answers is not in the document]
Comment (Mingbo-Lee) [reply docs/awesome-pets/papers/applications/ppml/fl/application.md:268]: I will remove it.

File: docs/awesome-pets/papers/applications/ppml/fl/incentive.md (hunk 2 above)
Comment (Mingbo-Lee) [reply docs/awesome-pets/papers/applications/ppml/fl/incentive.md:15]: Thank you for your suggestion.

File: docs/awesome-pets/papers/applications/ppml/fl/survey.md (hunk 3 above)
Comment (Mingbo-Lee) [reply docs/awesome-pets/papers/applications/ppml/fl/survey.md:13]: ``[Vertical Federated Learning](https://arxiv.org/pdf/2211.12814.pdf)`` is also a paper in arxiv and I think the structured literature review maybe can be accept as it was introduction above.

**Pull request closed without merging**